### ISSUE TITLE
MCMC needs manual seed setting

### ISSUE THREAD

**Issue by nickreich** [nickreich/coarseDataTools#45]
From Justin: It looks like MCMCpack is using some sort of default seed if we do not specify one. I can get different chains by setting my own seed. I think we could do an easy fix by doing a seed=as.numeric(Sys.time()) in the dic definition and then passing that along to MCMCpack

**Comment by jstockwin** [comment]
+1 on something like this happening. 
FWIW something like
```
  t <- as.numeric(Sys.time())
  seed <- 1e8 * (t - floor(t))
```
might be slightly better than what you said, as in your example the seed will increase linearly with time.

**Comment by jstockwin** [comment]
@nickreich This was fixed by #48 and can be closed.
